apps/avifenc.c

```diff
@@ -1052,17 +1105,17 @@ int main(int argc, char * argv[])
             lossless = AVIF_FALSE;
         }
 
-        if (!losslessColorQP) {
+        if (!losslessColorQuality) {
             fprintf(stderr,
-                    "WARNING: [--lossless] Color quantizer range (--min, --max) not set to %d. Color output might not be lossless.\n",
-                    AVIF_QUANTIZER_LOSSLESS);
+                    "WARNING: [--lossless] Color quality (--q) not set to %d. Color output might not be lossless.\n",
```

Comment:
-q then ?

Reply:
Done. I changed `--q` to `-q or --qcolor`.

Note: The long name for `-q` can be `--qcolor` or `--quality`. Which one do you prefer?